### DISCUSSION TITLE
Real Time Camera

### DISCUSSION THREAD

**Discussion by doudididodidadu** [tsunamayo/Starship-EVO#4248] [Suggestions]
So, the concept is that you take the view from a camera, and, you can link it to a screen that will show what the camera is seeing. This could be toggleable so that people who have strong enough computers can make that happen. Idk if it's possible but, it would be cool to have like a screen instead of a window

**Comment by cloudrambler** [comment]
Hi @doudididodidadu, This has been requested before. #3230
